rest-api/dao/questionnaire_response_dao.py

```diff
@@ -241,6 +244,15 @@ def _update_participant_summary(
             new_status = QuestionnaireStatus.SUBMITTED_NO_CONSENT
           elif code.value == CONSENT_FOR_DVEHR_MODULE:
             new_status = dvehr_consent
+          elif code.value == CONSENT_FOR_STUDY_ENROLLMENT_MODULE:
+            # set language of consent to participant summary
+            for extension in resource_json.get('extension', []):
+              if extension.get('url') == _LANGUAGE_EXTENSION and \
+                extension.get('valueCode') in LANGUAGE_OF_CONSENT:
```

Comment:
if extension.get('valueCode') is NOT in LANGUAGE_OF_CONSENT, we may want to log a warning here...

Reply:
done